client/components/review.js

```diff
@@ -125,10 +126,34 @@ class Review extends React.Component {
     );
   }
 
+  changed = (key) => {
+
+    if (this.props.latest.includes(key)) {
+      return LATEST;
+    }
+    else if (this.props.granted.includes(key)) {
+      return GRANTED;
+    }
+  }
+
+  changedBadge = change => {
+    switch (change) {
+      case LATEST:
+        return <span className="badge changed">changed</span>;
+      case GRANTED:
+        return <span className="badge">amended</span>;
+      default:
+        return null;
+    }
+  }
+
   render() {
     return (
       <div className="review">
         <h3>{this.props.label}</h3>
+        {
+          this.changedBadge(this.changed(`${this.props.prefix || ''}${this.props.name}`))
```

Comment:
Since all the arguments here are properties of `this` I'd just look up the key directly in `this.changedBadge()` rather than passing them around as arguments.

Reply:
That is, add 

```
const changed = this.changed(`${this.props.prefix || ''}${this.props.name}`)
``` 

to the top of `this.changedBadge()` and get rid of the argument.